### PR TITLE
Replacing GITHUB_TOKEN with PAT

### PR DESCRIPTION
**Pull request by datanerd**
Since the permissions for `GITHUB_TOKEN` are [limited](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret) to the repository that contains the workflow, it won't be possible to use `GITHUB_TOKEN` to access other private repositories. Instead, users would need to create and use a PAT to access other private repositories from a workflow.

### Why:

**Closes [issue link]**

### What's being changed:

Replacing `GITHUB_TOKEN` with `PERSONAL_ACCESS_TOKEN`

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by affrae** [comment]
Do we want to provide the GitHub Apps method as well?

**Comment by andrekolodochka** [comment]
> Do we want to provide the GitHub Apps method as well?

The GitHub App method is using a third party action. If it's ok to add a solution that depends on third party action into GitHub documentation - I'd love to. Can the GitHub docs team advise?

**Comment by janiceilene** [comment]
@andrekolodochka Thanks so much for opening a PR! I'll get this triaged for review ⚡

File: content/actions/reference/workflow-syntax-for-github-actions.md [hunk 1]
```diff
@@ -725,7 +725,7 @@ jobs:
 
 #### Example using action inside a different private repository than the workflow
 
-Your workflow must checkout the private repository and reference the action locally. You will need to generate a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token) and add it to your [repository](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) or [organisation](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization). (You will need to replace `PERSONAL_ACCESS_TOKEN` with the name of the secret you created.)
+Your workflow must checkout the private repository and reference the action locally. You will need to generate a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token) and add it to your [repository](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) or [organisation](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization). You will need to replace `PERSONAL_ACCESS_TOKEN` in the example below with the name of the secret you created.
```
Comment (affrae) [inline content/actions/reference/workflow-syntax-for-github-actions.md:728]: suggest replacing organisation with organization for spelling consistency in our docs

Comment (affrae) [reply content/actions/reference/workflow-syntax-for-github-actions.md:728]: Also - do we need to confirm the fact that the PAT will have the permissions of the user creating it? Eg may need a service account.

**Review by affrae — DISMISSED** [review]
I would suggest that we clarify what permissions the PAT would have and that a service account may be required.

File: content/actions/reference/workflow-syntax-for-github-actions.md (hunk 1 above)
Comment (Ice991) [reply content/actions/reference/workflow-syntax-for-github-actions.md]:
```suggestion
Your workflow must checkout the private repository and reference the action locally. You will need to generate a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token) and add it to your [repository](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository
Duplicate of #) or [organisation](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization). You will need to replace `PERSONAL_ACCESS_TOKEN` in the example below with the name of the secret you created.
```

Comment (andrekolodochka) [reply content/actions/reference/workflow-syntax-for-github-actions.md]: @Ice991 while those two links point to the same page, they are for two different sections on that page, one for an organisation, another is for a repository

**Review by lecoursen — DISMISSED** [review]
Thanks for this contribution! I'm going to apply a suggested change to more closely follow our style guide and will merge after that.

**Review by lecoursen — APPROVED** [review]

**Pull request merged**